ee/localserver/certs.go

```diff
@@ -1,6 +1,15 @@
 package localserver
 
+import (
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+
+	"github.com/lestrrat-go/jwx/jwk"
```

Comment:
Do we need to pull in this library? It feels a little sprawling, and I feel very -_- about third parties these days

Reply:
yea, it had quite a few dependencies, I dropped it and now just using std lib. More code to handle key conversions, but once separated, makes core logic code simpler.